cert/cert.go

```diff
@@ -127,15 +127,9 @@ func RecombineAndValidate(v Version, rawCertBytes, publicKey []byte, curve Curve
 
 	c, err := unmarshalCertificateFromHandshake(v, rawCertBytes, publicKey, curve)
```

Comment:
now that this function is just error checks, we could inline unmarshalCertificateFromHandshake?

Reply:
I have a slight preference to keep it here to reduce the complexity of the handshake functions.